visor/visor-config.md

```diff
@@ -0,0 +1,210 @@
+
+
+
+
+
+
+## *VisorConfigFile*
+Root of the config file
+
+
+### Fields
+
+<dl>
+<dt>
+	<code>maxNumberOfFirstConnectionRetries</code>  <strong>int</strong>  - optional
+</dt>
+
+<dd>
+
+Visor communicates with Core node via RPC API. This variable allows to specify
+how many times should Visor try to establish connection to Core node before the Visor process fails.
+The `maxNumberOfFirstConnectionRetries` is only taken to the account
+during the first start up of the Core node process - not restarts.
+
+
+
+Default value: <code>10</code>
+
+<blockquote>There is a 2 seconds delay between each try. Setting the max retry number to 5 means the Visor will try to establish
+5 connections times in 10 seconds.
+</blockquote>
+</dd>
+
+<dt>
+	<code>maxNumberOfRestarts</code>  <strong>int</strong>  - optional
+</dt>
+
+<dd>
+
+Visor at it's core starts and manages processes of provided binaries.
+This alows to define maximum number of restarts in case that any of
+the processes has failed before the Visor process fails.
```

Comment:
```suggestion
Visor starts and manages the processes of provided binaries.
This allows a user to define the maximum number of restarts in case any of
the processes have failed before the Visor process fails.
```